.gitignore

```diff
@@ -147,6 +147,8 @@ playwright-report
 test-results
 !tests/e2e/data/*.tar
 tests/e2e/.env
+!tests/cli/data/*.tar.gz
```

Comment:
Note: after merging, cli and e2e test data are expected to be combined eventually since they are really the same thing. So this will also get cleaned up along with it